src/main/java/com/treasuredata/client/model/TDFederatedQueryConfig.java

```diff
@@ -0,0 +1,140 @@
+package com.treasuredata.client.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.common.base.Objects;
```

Comment:
In `td-client-java`, we have abolished usage of `com.google.common.base.Objects`,

Could you rewrite the class without using `com.google.common.base.Objects`?

Reply:
Sure! Updated